src/ape/api/address.py

```diff
@@ -45,33 +75,86 @@ def __repr__(self) -> str:
         return f"<{self.__class__.__name__} {self.address}>"
 
     def __str__(self) -> str:
+        """
+        Convert this class to a ``str`` address.
+
+        Returns:
+            str: The stringified address.
+        """
+
         return self.address
 
     @property
     def nonce(self) -> int:
+        """
+        The number of transactions associated with the address.
+
+        Returns:
+            int
+        """
+
         return self.provider.get_nonce(self.address)
 
     @property
     def balance(self) -> int:
+        """
+        The total balance of the account.
+
+        Returns:
+            `int`
```

Comment:
are the ticks on purpose here? the others don't have it

Reply:
Accident!